server/bin/pbench-tarball-stats.py

```diff
@@ -175,10 +181,21 @@ def main(options):
     print(f"{invalid_cnt:18n} bad tar balls, total count")
     print(f"{invalid_size:18n} bad tar balls, total size")
 
-    print(
-        f"{len(good.keys()):18n} controllers with good tar balls (EC2 {ec2_cnt:n}, RH {rh_cnt:n})"
-    )
-    print(f"{good_cnt:18n} good tar balls, total count")
+    print(f"{len(good.keys()):18n} controllers with good tar balls (", end="")
+    satellites_ctrl = collections.Counter()
+    for good_ctrl in good.keys():
+        satellites_ctrl[get_sat_name(good_ctrl)] += 1
+    first = True
+    for name, total in sorted(satellites_ctrl.items()):
+        comma, first = ("", False) if first else (", ", False)
+        print(f"{comma}{name} {total:n}", end="")
+    print(")")
```

Comment:
It would be nice to skip printing the parentheses (and to skip executing everything else, here) if `len(good.keys() == 0`, e.g.:
```
    print(f"{len(good.keys()):18n} controllers with good tar balls", end="")
    if good:
        print(" (", end="")
        satellites_ctrl = collections.Counter()
        for good_ctrl in good.keys():
            satellites_ctrl[get_sat_name(good_ctrl)] += 1
        first = True
        for name, total in sorted(satellites_ctrl.items()):
            comma, first = ("", False) if first else (", ", False)
            print(f"{comma}{name} {total:n}", end="")
        print(")")
```

Ditto for lines 193-198.